offchain/danaswap-browser/src/DanaSwap/Browser/Home.purs

```diff
@@ -16,21 +25,249 @@ main = HA.runHalogenAff do
+                  ]
+              ]
+          , HH.div [ mkClass "column is-1" ]
+              [ HH.label [ HP.for "add-liquidity-asset-a", mkClass "label is-normal" ] [ HH.text a.name ]
+              ]
+          ]
+      , HH.div [ mkClass "columns is-mobile is-vcentered" ]
+          [ HH.div [ mkClass "column is-8-touch is-3-desktop" ]
+              [ HH.div [ mkClass "field" ]
+                  [ HH.div [ mkClass "control" ] [ HH.input [ HP.id "add-liquidity-asset-b", mkClass "input is-rounded" ] ]
+                  ]
+              ]
+          , HH.div [ mkClass "column is-1" ]
+              [ HH.label [ HP.for "add-liquidity-asset-b", mkClass "label is-normal" ] [ HH.text b.name ]
+              ]
+          ]
+      , HH.p [ mkClass "subtitle is-6 my-0" ]
+          [ HH.text "Liquidity Tokens: 0"
+          ]
+      , HH.p [ mkClass "subtitle is-6" ]
+          [ HH.text "Fee: 2.3 ADA"
+          ]
+      , HH.div [ mkClass "columns" ]
+          [ HH.div [ mkClass "column is-narrow" ]
+              [ HH.button [ HP.type_ ButtonSubmit, mkClass $ "button is-rounded is-medium is-responsive danaswap-btn-has-border" ] [ HH.text "submit" ]
+              ]
+          ]
+      ]
+
+    renderWithdrawLiquidityForm = HH.form [ mkClass "mt-6" ]
+      [ HH.div [ mkClass "columns is-mobile is-vcentered" ]
+          [ HH.div [ mkClass "column is-8-touch is-3-desktop" ]
+              [ HH.div [ mkClass "field" ]
+                  [ HH.div [ mkClass "control" ] [ HH.input [ HP.id "withdraw-liquidity-tokens", mkClass "input is-rounded" ] ]
+                  ]
+              ]
+          , HH.div [ mkClass "column is-1" ]
+              [ HH.label [ HP.for "withdraw-liquidity-tokens", mkClass "label is-normal" ] [ HH.text "Liquidity Tokens" ]
+              ]
+          ]
+      , HH.p [ mkClass "subtitle is-6 my-0" ]
+          [ HH.text "ADA Payout: 0"
+          ]
+      , HH.p [ mkClass "subtitle is-6 my-0" ]
+          [ HH.text "DANA Payout: 0"
+          ]
+      , HH.p [ mkClass "subtitle is-6 my-0" ]
+          [ HH.text "Slippage: 4%"
+          ]
+      , HH.p [ mkClass "subtitle is-6" ]
+          [ HH.text "Fee: 2.3 ADA"
+          ]
+      , HH.div [ mkClass "columns" ]
+          [ HH.div [ mkClass "column is-narrow" ]
+              [ HH.button [ HP.type_ ButtonSubmit, mkClass $ "button is-rounded is-medium is-responsive danaswap-btn-has-border" ] [ HH.text "submit" ]
+              ]
+          ]
+      ]
+
+mkClass :: forall (r :: Row Type) (i :: Type). String -> HP.IProp (class :: String | r) i
+mkClass = HP.class_ <<< HH.ClassName
+
+mkIcon :: forall slots m. String -> H.ComponentHTML Action slots m
+mkIcon icon = HH.span
+  [ mkClass "icon" ]
+  [ SE.svg [ role "none", width 16.0, height 16.0 ]
+      [ SE.use [ href ("/assets/images/font-awesome-sprite-solid.svg#" <> icon), fill $ Named "currentColor" ]
```

Comment:
We can skip all these `fill=currentColor` properties (here and in the EJS) if we tie in a CSS class such as
```css
.Icon {
   fill: currentColor;
   height: 1em;
}
```